src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const longTerm = 7;
+  const shortTerm = 3;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  const price = 40;
+
+  if (days >= longTerm) {
+    return days * price - bigDiscount;
+  }
+
+  if (days >= shortTerm) {
+    return days * price - smallDiscount;
+  }
+
+  return days * price;
```

Comment:
The return statement `return days * price` is correct for cases where no discounts apply. Ensure this aligns with the task requirements.